cloud_infra_center/ocp_upi/inventory.yaml

```diff
@@ -0,0 +1,89 @@
+# =================================================================
+# Copyright 2021 https://github.com/openshift/installer
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =================================================================
+
+# =================================================================
+# Licensed Materials - Property of IBM
+#
+# (c) Copyright IBM Corp. 2021 All Rights Reserved
+#
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+# =================================================================
+#
+# Change Summary: 
+# - Add bastion host
+# - Add auto allocated ip
+# - Modify items and delete unnecessary Ansible items
+
+all:
+  hosts:
+    localhost:
+      ansible_connection: local
+      ansible_python_interpreter: "{{ansible_playbook_python}}"
+
+      # User-provided values
+      vm_type: 'zvm'
+      disk_type: 'dasd'
+      openshift_version: '4.7'
+      openshift_minor_version: '0'
+
+      # The auto_allocated_ip default value is true, then IPs will be allocated from subnet range. 
+      # If you need specify IP addresses, set auto_allocated_ip to false.
+      auto_allocated_ip: true 
+      os_subnet_range: '172.26.0.0/16'
+      # By default, if the os_subnet_range is '172.26.0.0/16', the allocation pools will be 172.26.0.10-172.26.255.254
+      # The allocation pool can be configured by allocation_pool_start and allocation_pool_end
+      # allocation_pool_start: '172.26.0.2'
+      # allocation_pool_end: '172.26.0.10'
+      os_flavor_master: 'medium'
+      os_flavor_worker: 'large'
+      os_image_rhcos: 'rhcos'
+
+      # Name of the SDN.
+      os_networking_type: 'OpenshiftSDN'
+
+      # Number of provisioned Control Plane nodes
+      # 3 is the minimum number for a fully-functional cluster.
+      os_cp_nodes_number: 3
+
+      # Number of provisioned Compute nodes.
+      # 3 is the minimum number for a fully-functional cluster.
+      os_compute_nodes_number: 3
+
+    # Bastion DNS and HAProxy settings (Only for fixed IP address)
+    bastion:
+      ansible_ssh_host: 172.26.103.100
```

Comment:
I wonder why you chose to have hard coded IPs here instead of `{{ vars }}`

Reply:
The reason why hardcoded is that the items in inventory.yaml was an example for users. It's a good suggestion any how and I'll consider in the future.